.github/workflows/build.yml

```diff
@@ -0,0 +1,269 @@
+name: Build Channel
+on:
+  workflow_call:
+    inputs:
+      channel:
+        type: string
+        required: true
+      fake:
+        type: boolean
+        required: false
+        default: false
```

Comment:
These shouldn't be necessary anymore, and are admittedly ugly, but they were super helpful when debugging. I was planning to leave them in for now, just in case, but I can ditch them if you prefer.